<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConverterDialog</class>
 <widget class="QDialog" name="ConverterDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>433</width>
    <height>428</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Converter Settings</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>6</number>
   </property>
   <property name="rightMargin">
    <number>6</number>
   </property>
   <property name="bottomMargin">
    <number>6</number>
   </property>
   <item row="0" column="0" colspan="2">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Select files to convert:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="3">
    <widget class="QListWidget" name="itemsListWidget">
     <property name="alternatingRowColors">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Output file directory:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QLineEdit" name="outDirEdit"/>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Output file name:</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QLineEdit" name="outFileEdit"/>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Preset:</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QComboBox" name="comboBox"/>
   </item>
   <item row="5" column="0" colspan="3">
    <widget class="QCheckBox" name="checkBox">
     <property name="text">
      <string>Overwrite existing files</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0" colspan="3">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QToolButton" name="dirButton">
     <property name="text">
      <string>...</string>
     </property>
    </widget>
   </item>
   <item row="3" column="2">
    <widget class="QToolButton" name="fileNameButton">
     <property name="text">
      <string>...</string>
     </property>
    </widget>
   </item>
   <item row="4" column="2">
    <widget class="QToolButton" name="presetButton">
     <property name="text">
      <string>...</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ConverterDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>330</x>
     <y>408</y>
    </hint>
    <hint type="destinationlabel">
     <x>112</x>
     <y>425</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ConverterDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>393</x>
     <y>407</y>
    </hint>
    <hint type="destinationlabel">
     <x>383</x>
     <y>425</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
